### PR TITLE
html-css AllaSydoriak

### PR DESCRIPTION
**Pull request by AllaSydoriak**

### CODE REVIEW

File: homeworks/html-css/AllaSydoriak/css/reset.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+body{
```
Comment (denysdovhan) [inline homeworks/html-css/AllaSydoriak/css/reset.css:1]: Neat 👍

File: homeworks/html-css/AllaSydoriak/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+}
+nav {
+  align-items: center;
+  display: flex;
+  padding: 0 30px;
+}
+.links {
+  padding-right: 15px;
+}
+.links__a {
```
Comment (denysdovhan) [inline homeworks/html-css/AllaSydoriak/css/styles.css:20]: Is that [BEM](http://getbem.com/)?

Comment (denysdovhan) [reply homeworks/html-css/AllaSydoriak/css/styles.css:20]: If so, that's super cool 👍

File: homeworks/html-css/AllaSydoriak/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (denysdovhan) [inline homeworks/html-css/AllaSydoriak/package.json:1]: 👍

File: homeworks/html-css/AllaSydoriak/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  "name": "homework",
```
Comment (denysdovhan) [inline homeworks/html-css/AllaSydoriak/package.json:2]: Consider also adding `"private": true` field.

**Review by denysdovhan — APPROVED** [review]

File: homeworks/html-css/AllaSydoriak/css/styles.css (hunk 2 above)
Comment (AllaSydoriak) [reply homeworks/html-css/AllaSydoriak/css/styles.css:20]: Yes, smth like that :smile:

File: homeworks/html-css/AllaSydoriak/package.json (hunk 4 above)
Comment (AllaSydoriak) [reply homeworks/html-css/AllaSydoriak/package.json:2]: Ok, i'll pay attention to it, thanks

**Pull request merged**